strings/names/first_male.txt

```diff
@@ -1,743 +1,872 @@
+	Dario
+	Darius
+	Darpan
+	Darren
+	David
+	Davion
+	Davis
+	Dawson
+	Dayton
+	Dean
+	Deandre
 	Deangelo
+	Declan
 	Denholm
+	Dennis
+	Derek
+	Derrick
 	Desmond
+	Devaj
 	Devin
+	Devon
+	Diego
+	Dillon
+	Dinh
 	Dirk
+	Dmitriy
 	Dominic
+	Dominick
+	Donald
+	Dong-Hyun
 	Donny
+	Donovan
+	Dorian
+	Douglas
+	Drake
+	Drew
 	Driscoll
 	Duke
 	Duncan
+	Dustin
+	Dylan
+	Easton
+	Eddie
 	Edgar
+	Edmundo
+	Eduardo
+	Edward
+	Edwin
+	Eggwrap
+	Eli
+	Elias
+	Elijah
 	Eliot
 	Eliott
+	Elliot
+	Elliott
 	Elric
 	Elspet
 	Elwood
+	Emanuel
+	Emerson
+	Emiliano
+	Emilio
 	Emmanuel
-	Eggwrap
+	Enrique
+	Eric
+	Erick
+	Erik
+	Erik
+	Ernesto
+	Esteban
+	Ethan
+	Evan
+	Everett
+	Ezekiel
+	Ezra
+	Fabian
+	Faheem
+	Faisal
+	Fayez
+	Felix
 	Fenton
+	Fernando
+	Finn
 	Fitz
 	Flick
 	Flint
 	Flip
 	Francis
+	Francisco
 	Frank
 	Frankie
+	Franklin
 	Fridge
 	Fulton
+	Gabriel
+	Gael
+	Gage
 	Gannon
 	Garret
+	Garrett
 	Gary
+	Gavin
+	George
+	Gerardo
+	Giovanni
 	Goddard
 	Godwin
 	Goodwin
 	Gordon
+	Grady
 	Graeme
+	Graham
 	Grandpa
+	Grant
 	Gratian
+	Grayson
+	Gregory
 	Grendel
+	Greyson
+	Griffin
+	Guillermo
+	Guneet
+	Gunnar
+	Gunner
+	Guohua
+	Guoqiang
+	Gustavo
+	Hafeez
+	Hai
+	Halvar
+	Hampus
 	Han
+	Han
+	Hansuke
+	Haoyu
 	Happy
+	Harish
+	Harrison
 	Harry
 	Hartley
+	Haru
+	Haruto
 	Harvey
+	Hashem
+	Hassan
+	Hatem
+	Hayden
+	Hector
 	Henderson
+	Henry
+	Hinata
 	Holden
 	Homer
 	Hope
 	Horatio
+	Hudson
 	Huffie
-	Hungry
 	Hugo
+	Hungry
+	Hunter
+	Ian
 	Irvine
+	Isaac
+	Isaiah
+	Isaias
+	Ishaan
+	Ishir
+	Isiah
+	Ismael
+	Israel
+	Issac
+	Ivan
+	Ivan
+	Izaiah
+	Jace
+	Jack
+	Jackson
 	Jacob
+	Jacoby
+	Jaden
+	Jadon
+	Jaiden
+	Jaime
 	Jake
+	Jakob
+	Jalen
+	Jamal
 	Jamar
+	Jamari
+	Jamarion
+	James
+	Jameson
 	Jamie
 	Jamison
 	Janel
+	Jared
+	Jason
+	Jasper
+	Javier
+	Javon
+	Jaxon
+	Jaxson
+	Jay
+	Jayce
+	Jayden
 	Jaydon
 	Jaye
+	Jaylen
 	Jayne
+	Jayson
 	Jean-Luc
 	Jeb
 	Jed
+	Jeffrey
 	Jemmy
+	Jeong-Hoon
+	Jeremiah
+	Jeremy
 	Jermaine
 	Jerrie
+	Jerry
+	Jesse
+	Jesus
+	Ji-Hoon
+	Jianguo
+	Jianjun
+	Jianping
 	Jim
+	Jimmy
+	Jiro
+	Jivin
 	Joachim
+	Joaquin
+	Joe
+	Joel
 	Joey
-	Johnathan
+	Johan
 	John
+	Johnathan
 	Johnny
-	Jonathon
+	Jonah
+	Jonas
+	Jonathan
+	Jonathon
+	Jordan
+	Jorge
+	Jose
+	Joseph
 	Josh
+	Joshua
 	Josiah
+	Josue
+	Juan
+	Juan
+	Judah
+	Jude
+	Julian
+	Julio
+	Julius
+	Jun
+	Junjie
+	Justice
+	Justin
+	Kade
+	Kaden
+	Kai
+	Kai
+	Kaiden
+	Kaito
+	Kale
+	Kaleb
+	Kameron
+	Kane
+	Kao
+	Kareem
+	Karson
+	Kayden
+	Keagan
+	Keaton
+	Keegan
+	Keiji
+	Keith
+	Kellen
+	Kelvin
 	Kennard
+	Kenneth
+	Kevin
 	Keziah
+	Khalil
+	Khalil
+	Khemkhaeng
+	Kija
+	Kingston
+	Kobe
+	Koji
+	Kolton
+	Krishiv
+	Kristopher
+	Kyle
+	Kyler
+	Kyong
+	Lance
+	Landen
 	Lando
+	Landon
+	Landyn
+	Lane
 	Lanny
+	Larry
 	Launce
+	Lawrence
+	Lei
+	Leif
 	Leland
 	Lennox
 	Lenny
+	Leo
 	Leonard
+	Leonardo
+	Leonel
 	Leroy
+	Levi
+	Li
+	Liam
 	Lief
+	Lijun
+	Lincoln
 	Linden
 	Linton
+	Liu
+	Logan
 	Lorde
+	Lorenzo
 	Loreto
 	Lou
+	Louis
+	Luca
 	Lucas
+	Luhan
+	Luis
+	Luis
```

Comment:
luis duplicated

Reply:
removed duplicates again. Error on my part from adding and forgetting to set the additions to ignore duplications